ext/datadog_profiling_native_extension/native_extension_helpers.rb

```diff
@@ -104,7 +104,7 @@ def self.libdatadog_folder_relative_to_ruby_extensions_folders(
         # the Gem.dir.
         expected_ruby_extensions_folders = [
           "#{gem_dir}/extensions/platform/extension_api_version/datadog_version/",
-          "#{gem_dir}/bundler/gems/extensions/platform/extension_api_version/datadog_version/",
+          "#{gem_dir}/bundler/gems/extensions/platform/extension_api_version/datadog_version/"
```

Comment:
I think all the cop changes looks super reasonable!
Except for removing the trailing comma from multi-line lists, like this one.

This guarantees that diffs for this list have a useless diff line on the trailing element, just adding a comma.

It's a small thing, but you always makes me slightly confused when I'm reviewing changes and I don't know if someone changed the last element of the list or not. Specially, for long lines, like `"#{gem_dir}/bundler/gems/extensions/platform/extension_api_version/datadog_version/"`, I have to inspect the whole string, instead of it simply never appearing in the diff, because in fact it did not change at all.

Reply:
> accept it as-is

I want to yield such power!

Reply:
I see this was already reported: https://github.com/standardrb/standard/issues/611

Reply:
Yeah the comma thing is the most annoying one ever (with the "double-quote strings everywhere" as a close second). I mean beyond diffs, in editors "duplicate line" is a very useful thing.

I really don't understand it, I recall their argument being mostly "well you shouldn't care because it's auto formatted on save".

A workaround is:

- disable the rule at the standard level (standard.yml)
- enforce the chosen rule at the rubocop level

Reply:
So it's clear there are ways of bending standardrb to our will on this one.

My question is: do we want to?

I'm leaning towards soft-no just because it goes ~slightly against the spirit of standardrb, but being a soft-no, I'm happy to go in the other direction.

Reply:
There's value in not quibbling. See, this PR seems blocked because of this? ;)

We can revisit later if we find it very annoying.